### ISSUE TITLE
typo in upgrade notes for 2.14

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1155]
Originally reported of Google Code with ID 1099.

```
http://docs.ganeti.org/ganeti/2.14/html/upgrade.html

In the 2.13 section,  gnt-cluster renew-crypt should be gnt-cluster renew-crypto.
```
Originally added on 2015-06-08 18:06:53 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-06-09 08:17:17 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit aaecd556a9b340d5727c1dea944d145c61317138
Author: Klaus Aehlig <aehlig@google.com>
Date:   Tue Jun 9 10:17:54 2015 +0200

    fix typo in UPGRADE notes
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>


Fixed in 2.13 where the typo was introduced; will be fixed in the
higher branches with the next merge up.
```
Originally added on 2015-06-09 08:26:41 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.13
